### DISCUSSION TITLE
Ability to bulk download from user profile categories

### DISCUSSION THREAD

**Discussion by brainage04** [ppy/osu#27809] [Ideas] [answered]
I find myself frequently downloading all of the maps from specific categories on user profiles within osu!lazer, specifically the Best Performance categories of top players and especially from Ranked/Loved/Guest Participation Beatmaps categories of mappers that I like.

I think it would be very helpful to add a button next to the category name which started downloading all of the maps in that category, perhaps with a pop-up requiring the user to specify how many maps they want to download starting from the first map (especially for larger categories like Favourite Beatmaps, which can contain 500+ maps).

Would this be possible? I'm not sure if downloading 100+ beatmaps in succession would trigger some kind of rate limit.

**Comment by Walavouchey** [comment, marked as answer]
similar features have been proposed in

https://github.com/ppy/osu/issues/2351
https://github.com/ppy/osu/issues/10192
https://github.com/ppy/osu/discussions/13204
https://github.com/ppy/osu/discussions/15205
https://github.com/ppy/osu/discussions/23803

this is very much not happening as has been stated each time
